<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Vishwajit" failures="2" tests="2" name="Test(failed)" time="0.069" errors="0" timestamp="2024-09-24T22:39:33 IST">
  <testcase classname="ReTryTestCases.ReTestCase" name="TestCase1" time="0.016">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at ReTryTestCases.ReTestCase.TestCase1(ReTestCase.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- TestCase1 -->
  <testcase classname="ReTryTestCases.ReTestCase" name="TestCase3" time="0.003">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at ReTryTestCases.ReTestCase.TestCase3(ReTestCase.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- TestCase3 -->
  <testcase classname="ReTryTestCases.ReTestCase" name="TestCase2" time="0.0">
    <ignored/>
  </testcase> <!-- TestCase2 -->
</testsuite> <!-- Test(failed) -->
